src/blocks/Balances.vue

```diff
@@ -105,6 +105,12 @@ export default Vue.extend({
     };
   },
   computed: {
+    isRinkeby(): boolean {
+      return this.$store.getters["zk-provider/network"] === "rinkeby";
+    },
+    etherscanLink(): string {
```

Comment:
Compose it into one computed. I mean just use this.$store.getters... inside the etherscanLink().
And assure ropsten check as well. Also, I think it would be better to point the link on the logged in wallet: ```https://etherscan.io/address/0x2f0297b26ec878bff6a0e37f695d72b7c205d812```
Then for the ropsten / rinkeby it would be ```https://ropsten.etherscan.io/address/0x2f0297b26ec878bff6a0e37f695d72b7c205d812```
/ ```https://etherscan.io/address/0x2f0297b26ec878bff6a0e37f695d72b7c205d812```